device/bgp/bgpsrtesegment.yaml

```diff
@@ -3,47 +3,490 @@ components:
+          type: integer
+          minimum: 0
+          maximum: 255
+          default: 0
+        ipv6_node_address:
+          description: >-
+            IPv6 address representing a node.
+          type: string
+          format: ipv6
+        srv6_sid:
+          description: >-
+            Optional SRv6 SID.
+          type: string
+          format: ipv6
+        srv6_sid_endpoint_behavior:
+          description: >-
+            Optional SRv6 Endpoint Behavior and SID Structure.
+          $ref: '#/components/schemas/BgpSrte.SRv6SIDEndpointBehaviorAndStructure'
+
+    BgpSrte.SegmentJTypeSubTlv:
+      description: >-
+        Type J: IPv6 Address, Interface ID for local and remote
+        pair for SRv6 with optional SID.
+      type: object
+      required: [local_ipv6_node_address,remote_ipv6_node_address]
+      properties:
+        flags:
+          description: >-
+            One octet of segment flags for V-Flag, A-Flag, S-Flag, B-Flag etc. as defined in
+            https://datatracker.ietf.org/doc/html/draft-ietf-idr-segment-routing-te-policy-13#section-2.4.4.2.12
+          type: integer
+          minimum: 0
+          maximum: 255
+          default: 0
+        sr_algorithm:
+          description: >-
+            specifying SR Algorithm when when A-Flag as defined in above flags.
+          type: integer
+          minimum: 0
+          maximum: 255
+          default: 0
+        local_interface_id:
+          description: >-
+            Local Interface ID: The Interface Index as defined in [RFC8664].
+          type: integer
+          minimum: 0
+          maximum: 2147483647
+          default: 0
+        local_ipv6_node_address:
+          description: >-
+            IPv6 address representing a node.
+          type: string
+          format: ipv6
+        remote_interface_id:
+          description: >-
+            Local Interface ID: The Interface Index as defined in [RFC8664].
+          type: integer
+          minimum: 0
+          maximum: 2147483647
+          default: 0
+        remote_ipv6_node_address:
+          description: >-
+            IPv6 address representing a node.
+          type: string
+          format: ipv6
+        srv6_sid:
+          description: >-
+            Optional SRv6 SID.
+          type: string
+          format: ipv6
+        srv6_sid_endpoint_behavior:
+          description: >-
+            Optional SRv6 Endpoint Behavior and SID Structure.
+          $ref: '#/components/schemas/BgpSrte.SRv6SIDEndpointBehaviorAndStructure'
+      
+    BgpSrte.SegmentKTypeSubTlv:
+      description: >-
+        Type K: IPv6 Local and Remote addresses for SRv6 with
+        optional SID.
+      type: object
+      required: [local_ipv6_address,remote_ipv6_address]
+      properties:
+        flags:
+          description: >-
+            One octet of segment flags for V-Flag, A-Flag, S-Flag, B-Flag etc. as defined in
```

Comment:
One octet bitmap for flags including V-Flag, A-Flag, S-Flag, B-Flag etc. as defined in